dockers/hypervisor/Dockerfile

```diff
@@ -2,9 +2,17 @@ FROM isard/alpine-pandas:1.0.0
 MAINTAINER isard <info@isard.com>
 
 RUN pip3 uninstall pandas pytz python-dateutil six -y
+RUN sed -i -e 's/v3\.8/v3.9/g' /etc/apk/repositories
+RUN apk update
+RUN apk upgrade --available
 
 RUN apk --no-cache add qemu-system-x86_64 libvirt netcat-openbsd libvirt-daemon dbus polkit qemu-img
-RUN ln -s /usr/bin/qemu-system-x86_64 /usr/bin/qemu-kvm
+RUN apk --no-cache add alsa-utils alsa-lib qemu-audio-alsa
+ADD dockers/hypervisor/asoundrc /root/.asoundrc
```

Comment:
We should try to switch all the `ADD` statements from the Dockerfiles to `COPY`. It's a best practice, you can check it [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)

Reply:
Sure, and as I see threre are audio alsa packages not needed. I'll push again this branch!